code/datums/movement/atom_movable.dm

```diff
@@ -8,6 +8,7 @@
 
 // Movement relay
 /datum/movement_handler/move_relay/DoMove(var/direction, var/mover)
+	set waitfor = FALSE
```

Comment:
This will break various movement-related stuff, like falling. Please don't do this.